Makefile

```diff
@@ -71,11 +71,13 @@ else
   TAG?=latest 
 endif
 
+# Image Repo to use when pushing/pulling any image
+IMG_REPO?=
 # Image URL to use for building/pushing of the operator
-OPERATOR_IMG ?= verticadb-operator:$(TAG)
+OPERATOR_IMG ?= $(IMG_REPO)verticadb-operator:$(VERSION)
```

Comment:
Can we keep using `$(TAG)`, but lets set it to equal `$(VERSION)`?  These is some conditional logic to set `$(TAG)` based on whether we are running kind.  That can be removed entirely.